src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +40,6 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        return numbers.stream().filter(number -> number > 3).map(x -> 2 * x).reduce(0, (x, y) -> x + y);
```

Comment:
```suggestion
        return numbers.stream()
            .filter(x -> x > 3)
            .map(x -> x * 2)
            .reduce(0, Integer::sum);
```
- 앞선 내용처럼 한 줄에 한 개의 메소드만 사용해주세요!
- 변수 명을 통일해주세요!
- `Integer::sum` 을 이용해도 된답니다!